CIP-????/README.md

```diff
@@ -0,0 +1,247 @@
+---
+CIP: ?
```

Comment:
```suggestion
CIP: 45
```

Reply:
Nice CIP-0045 sounds wonderful to me 😊 ! Meanwhile the following points along the implementation plan has been succeeded: 

1. The [bugout reimplementation](https://github.com/fabianbormann/meerkat) into a typescript and webpack 5 friendly lib has been done 
2. The [cardano-peer-connect lib](https://github.com/fabianbormann/cardano-peer-connect) has been created to simplify the implementation of CIP-0045 for dApp and wallet app developers
3. The [example implementation](https://github.com/fabianbormann/cip-xxxx-demo-implementation) has been refactored so that it now uses the peer-connect lib 

During the implementation it turned out that a we need to invest some more time into:

1. Testing different scenarios. For example the communication with a network setup using a symmetric NAT
2. If the dapp and the wallet  app runs on the same device or in the same subnet (without NAT) we can simplify the communication establishment
3. Heartbeats, reconnection and a dynamic update of the [WebTorrent tracker list](https://github.com/ngosang/trackerslist) would be useful to make the communication more solid

The next steps would be to provide a reference implementation within a cip-0030 compliant wallet app.